### DISCUSSION TITLE
PnP Search Results - Not returning results for external users

### DISCUSSION THREAD

**Discussion by AlexanderBld** [microsoft-search/pnp-modern-search#2949] [General]
**Version used**
Ex: 4.8

**Describe the bug**
When logged in as an external user, **PnP - Search Results** webpart does not return results, either using SharePoint Search or Microsoft Search.

**To Reproduce**
Detailed steps to reproduce the behavior:
1. Create a new Site Page
2. Add the PnP Search box webpart
3. Add the PnP Search results web part
4. Select SharePoint Search or Microsoft Search
5. Select People as layout
6. Select PnP Search box as source

**Expected behavior**
Logged in as an internal user behavior is as expected, showing results when a search is performed. On the other hand, logged in as an external user no results are displayed.

**Desktop:**
 - Chrome, Edge, Firefox

**Comment by kasperbolarsen** [comment]
Hi @AlexanderBld, could you please check if this behavior is the same in the classic search and/or Microsoft Search?

**Comment by kasperbolarsen** [comment]
Wait a second,  external are not allowed to search in the LocalPeopleIndex, only LocalSharePointResults. 

It sounds like it works as designed

**Reply by wobba** [reply]
Correct. Exposing all AAD/UPA accounts to external people is not good for privacy reasons.
